printer_zpl2/README.rst

```diff
@@ -15,6 +15,7 @@ See below for more details.
 Installation
 ============
 
+This module requires zpl2 >= version 1.1.
```

Comment:
Instead of just noting the requirement, we need to state this as an upgrade note. Users upgrading pre-existing installations are the ones that this affects, not new users. As such, I recommend:

```
**Upgrade Note:** If upgrading from a version prior to 10.0.2.0.0, you must make sure to update the `zpl2` Python package to at least version 1.1.
```

Also, please bump the module version by a major version: 10.0.2.0.0